web/modules/custom/default_content_config/content/taxonomy_term/04c4f241-964c-44ae-a469-8c65cc7fc4cd.yml

```diff
@@ -0,0 +1,28 @@
+_meta:
+  version: '1.0'
+  entity_type: taxonomy_term
+  uuid: 04c4f241-964c-44ae-a469-8c65cc7fc4cd
+  bundle: hcd_glossary
+  default_langcode: en
+default:
+  status:
+    -
+      value: true
+  name:
+    -
+      value: 'Qualitative research'
+  description:
+    -
+      value: '<p><span style="font-family:&quot;Aptos&quot;,sans-serif;font-size:12.0pt;">Primarily exploratory research used to gain an understanding of underlying reasons, opinions, and motivations. It provides insights into the problem or helps to develop ideas or hypotheses for potential quantitative research.</span></p>'
```

Comment:
@cathybaptista looks like you grabbed the HTML output from the hugo site and not the raw descriptions from `static/glossary/hcd-glossary.json` which don't have any markup tags. Can you clean up the descriptions? Fastest is probably to cut and paste from the JSON instead of manually removing tags.

Reply:
This applies to all terms not just this one.

Reply:
Looking at the .json file for the terms, it looks like they do allow some HTML. They have <em> tags in one term. However, the display does not allow for <p> tags. We solve this by using the 'Single Inline HTML' format filter. That will allow very simple HTML filters but remove any break or paragraph tags.
It does not look like you can set the format filter for the description field of terms using the GUI. So I would either:
* Alter the term edit form and set the format filter manually.
* Or hide the description field in form display and add your own field.

As for the issue with pasting, I get how you're getting <p> tags, because those get added automatically, but I don't understand the <span> tags.
`<span style="font-family:&quot;Aptos&quot;,sans-serif;font-size:12.0pt;">`. Looking at the dom of https://digital.gov/guides/hcd/introduction/, I don't see that anywhere.

Reply:
@cathybaptista the <p> tags are expected in the export, they are stripped out on display. Ie, they must be run through the format filter to remove.